screens/DayScreen.tsx

```diff
@@ -43,37 +47,69 @@ const DayScreen: React.FC<DayInfo> = (dayInfo: DayInfo) => {
   const todayIsDay = checkDay(dayInfo.day);
   const now = new Date();
 
-  const isNow = (timeString: string) => {
+  const isNow = (timeString: TimeRange | string) => {
     if (!todayIsDay) return false;
 
-    if (timeString.includes('t.e.m.') && !timeString.includes('...')) {
-      const split = timeString.split('t.e.m. ');
-      const startTime = split[0].split(':');
-      const endTime = split[1].split(':');
-      //Create dummy dates
-      const start = new Date(
-        now.getFullYear(),
-        now.getMonth(),
-        now.getDate(),
-        parseInt(startTime[0]),
-        parseInt(startTime[1]),
-        0,
-      );
-      const end = new Date(
-        now.getFullYear(),
-        now.getMonth(),
-        now.getDate(),
-        parseInt(endTime[0]),
-        parseInt(endTime[1]),
-        0,
-      );
-
-      return now >= start && now <= end;
+    //Backwards compatibility
```

Comment:
Ben over het algemeen wel fan van dingen wat opsplitsen in aparte methodes als er veel logica en/of if/else statements zijn. Maakt het iets minder mentaal belastend om te snappen waarover het precies gaat